apps/condo/domains/acquiring/constants/gql.js

```diff
@@ -6,20 +6,43 @@ const FEE_DISTRIBUTION_SCHEMA_INPUT_NAME = `[${FEE_DISTRIBUTION_INPUT_NAME}!]`
 const SETTINGS_FIELD_NAME = 'SettingsField'
 const SETTINGS_INPUT_NAME = 'SettingsFieldInput'
 
+const RECIPIENT_FIELDS_DEFINITION = {
+    name: 'String',
+    bankName: 'String',
+    territoryCode: 'String',
+    offsettingAccount: 'String',
+    tin: 'String!',
+    iec: 'String',
+    // TODO(YEgorLu): DOMA-10442 rename to routingNumber
+    bic: 'String!',
+    bankAccount: 'String!',
+    classificationCode: 'String',
+}
+
 const BILLING_RECEIPT_FIELD_NAME = 'BillingReceiptsField'
 const BILLING_RECEIPT_INPUT_NAME = 'BillingReceiptsFieldInput'
 const BILLING_RECEIPTS_SCHEMA_FIELD_NAME = `[${BILLING_RECEIPT_FIELD_NAME}!]`
 const BILLING_RECEIPTS_SCHEMA_INPUT_NAME = `[${BILLING_RECEIPT_INPUT_NAME}!]`
 
+const AMOUNT_DISTRIBUTION_FIELD_NAME = 'AmountDistributionField'
+const AMOUNT_DISTRIBUTION_INPUT_NAME = 'AmountDistributionFieldInput'
+const AMOUNT_DISTRIBUTION_SCHEMA_FIELD = `[${AMOUNT_DISTRIBUTION_FIELD_NAME}!]`
+const AMOUNT_DISTRIBUTION_SCHEMA_INPUT = `[${AMOUNT_DISTRIBUTION_INPUT_NAME}!]`
```

Comment:
Why not in field file?

Reply:
There was a problem with cyclic inclusions. But now it is gone. Constants were moved to a file containing the field definition.